pkg/swap/swap.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
Ok, I think I got it sh-ified. Certainly it will be good enough for now. When we git `mkswap` and `swapon` as golang calls, I will look at rewriting this as well, and then it is a single binary.